Geometry2d/package.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
```

Comment:
Shouldn't we not need to make separate packages, because we're not using colcon?

Reply:
Yeah that's true. I'm not sure if we still want to have the separate packages though?

Reply:
What do you mean by this?

Reply:
nvm. I'll remove it